packages/test/src/helpers.ts

```diff
@@ -217,6 +217,7 @@ export class TestWorkflowEnvironment extends RealTestWorkflowEnvironment {
 // Some of our tests expect "default custom search attributes" to exists, which used to be the case
 // in all deployment with support for advanced visibility. However, this might no longer be true in
 // some environement (e.g. Temporal CLI). Use the operator service to create them if they're missing.
+// TODO: register search attributes when creating test environment (#1624)
```

Comment:
Not sure what is the meaning of this FIXME. Remove?

Reply:
It was a reminder to myself to clean this up, removed.